WcaOnRails/app/models/competition.rb

```diff
@@ -177,6 +177,8 @@ class Competition < ApplicationRecord
   validates :cityName, :countryId, :venue, :venueAddress, :latitude, :longitude, presence: true, if: :confirmed_or_visible?
   validates :external_website, presence: true, if: -> { confirmed_or_visible? && !generate_website }
 
+  validates :registration_open, :registration_close, presence: { message: I18n.t('simple_form.required.text') }, if: :registration_period_required?
```

Comment:
I'm kind of surprised we need a custom message here (we don't have one for `external_website` above). Does something ugly/bad happen if we remove it?

Reply:
I was quite surprised seeing it too and ignored it, but then a test failed indicating that this message is indeed different than the default presence check. In the context of dates *is required* is probably more clear than *can't be blank*. I don't have a strong opinion here =)

Reply:
Ahh, gotcha. Yeah, this seems fine to me!